.travis.yml

```diff
@@ -5,7 +5,10 @@ python:
 install:
   - pip install pandas
   - pip install numpy
+  - pip install codecov
   # The latest version of doit supporting Python 2 is 0.29.0
   - test $(python -c 'import sys;print(sys.version_info[0])') -eq 2 && pip install 'doit==0.29.0' || pip install doit
 script:
-  - doit test
\ No newline at end of file
+  - doit test
```

Comment:
this does coverage calculation, but is still called test. Maybe rename the task?

Reply:
Measuring the code coverage of tests (i.e. test coverage) is a means to see which parts of the program are tested how well. So in my option, coverage measurement is just an additional enhancement of tests. Once you can measure your test coverage as easy as we can now, I don't see why you would ever want to run your tests without measuring their code coverage.

To me, our tests now became tests that report their code coverage. And thats why I think that it should still be named `test`.

Does this make sense?